src/constants.ts

```diff
@@ -7,3 +9,24 @@ export const OPENSEA_BASE_NFTS_URL = `${OPENSEA_BASE_URL}/assets`;
 export const MAGICEDEN_BASE_URL = 'https://magiceden.io';
 export const MAGICEDEN_BASE_COLLECTIONS_URL = `${MAGICEDEN_BASE_URL}/marketplace`;
 export const MAGICEDEN_BASE_NFTS_URL = `${MAGICEDEN_BASE_URL}/item-details`;
+
+const evmLabels = {
+    collectionId: "Contract Address",
+    tokenId: "Token ID",
+    tokenStandard: "ERC721"
```

Comment:
a nit without a solution, but `tokenStandard` seems to stand out as being a common _value_ instead of _label_? any way we can make a distinction?

Reply:
Hmmm, maybe "label" is the wrong word then. We could call them all `chainConceptConstants` or something?

Reply:
actually maybe tokenStandard doesnt even belong here - it could be brought into server/, as even within EVM chains, we could have 721s or 1155s on the site

Reply:
I like the idea of removing `tokenStandard`, especially since the value should technically come from our DB

Reply:
it does, the only thing that consumes this is init.ts actually, def just belongs in the server